common/acm/templates/provision/clusterpool.yaml

```diff
@@ -0,0 +1,82 @@
+{{- range .Values.clusterGroup.managedClusterGroups }}
+{{- $group := . }}
+apiVersion: cluster.open-cluster-management.io/v1beta1
+kind: ManagedClusterSet
+metadata:
+  annotations:
+    cluster.open-cluster-management.io/submariner-broker-ns: {{ .name }}-broker
+  name: {{ .name }}
+spec:
+  clusterSelector:
+    selectorType: LegacyClusterSetLabel
```

Comment:
Are there no other ways to do this? This selectorType is slated to be removed in: https://issues.redhat.com/projects/ACM/issues/ACM-1521